src/lib/util.ts

```diff
@@ -231,6 +231,20 @@ export function parseToNumberString(num: number, getString: GetStringFunction):
 	return `${num}`
 }
 
+export const localeList = ["ar", "bg", "cs", "da", "de", "el", "empty", "en", "enpt", "es", "esmx", "espe", "fa", "fi", "fil", "fr", "he", "hi", "hu", "ib", "id", "it", "ja", "ko", "ms", "nl", "no", "pl", "pt", "ptbr", "ro", "ru", "sv", "th", "tr", "ua", "zhcn", "zhtw"]
+
+export function discordLocaleToBotLocale(discordLocale: string): string {
+	if (discordLocale === "en-US" || discordLocale === "en-GB") discordLocale = "en"
+	else if (discordLocale === "es-ES") discordLocale = "es"
+	else if (discordLocale === "sv-SE") discordLocale = "sv"
+	else if (discordLocale === "uk") discordLocale = "ua"
```

Comment:
Why the locale of Ukrainian is `ua`? `ua` is the country code and `uk` is the language locale.

Reply:
I just changed it so it matches the strings/ folder. @ImRodry?

Reply:
Yeah that’s correct but this PR got me thinking about changing the locales we use since initially we used these as they were easier to type but now we have autocomplete so we don’t need that anymore, might look into that later today

Reply:
Could this be done outside of this PR this time? 🥲

Reply:
Yes of course, you’re just gonna get yet another merge conflict!